harvester_e2e_tests/apis/test_hosts.py

```diff
@@ -131,8 +80,11 @@ def test_update_node(api_client):
     )
 
     # For teardown
-    _ = api_client.hosts.update(node['id'],
-                                dict(metadata=dict(annotations=original_annotations)))
+    sleep(1)  # to prevent update too fast cause code 409 conflict: 'object has been modified'
```

Comment:
Do we have better way to avoid 409 conflict, like `get` after `update` to make sure the object is update?

Reply:
I don't have any better way to prevent resource conflict for now, this more like a temporarily fix because it will not always happen in the test case.

The problem of conflict is, we update host's information and do few check in million seconds, then revert (update) it to the original information.  when calling `hosts.update`, API will do `hosts.get` internally to make sure the resource is latest, but as the result, it will not always work (then hit resource conflict).